packages/paste-core/components/topbar/stories/userDialog.stories.tsx

```diff
@@ -0,0 +1,120 @@
+import * as React from 'react';
+import type {StoryFn} from '@storybook/react';
+import {UserIcon} from '@twilio-paste/icons/esm/UserIcon';
+import {Button} from '@twilio-paste/button';
+import {Box} from '@twilio-paste/box';
+import {CustomizationProvider} from '@twilio-paste/customization';
+import {useTheme} from '@twilio-paste/theme';
+
+import {
+  useUserDialogState,
+  UserDialog,
+  UserDialogUserInfo,
+  UserDialogUserName,
+  UserDialogUserEmail,
+  UserDialogContainer,
+} from '../src';
+
+// eslint-disable-next-line import/no-default-export
+export default {
+  title: 'Components/Topbar',
+};
+
+export const BasicUserDialog: StoryFn = () => {
+  return (
+    <UserDialogContainer name="User Name" icon={UserIcon} baseId="i-am-user-dialog">
+      <UserDialog aria-label="user menu" data-testid="basic-user-dialog">
+        <UserDialogUserInfo>
+          <UserDialogUserName>Name</UserDialogUserName>
+          <UserDialogUserEmail>email@email.com</UserDialogUserEmail>
+        </UserDialogUserInfo>
+      </UserDialog>
+    </UserDialogContainer>
+  );
+};
```

Comment:
question: as we're extending the avatar do we need an image or `src` version to prove it works?

Reply:
yup good catch